### ISSUE TITLE
XashDS Server won't show up on public server list.

### ISSUE THREAD

**Issue by lilmayofuksu** [FWGS/xash3d-fwgs#155]
The servers opened with dedicated build of the new engine won't show up in masterserver. When I tried hosting masterserver locally and adding as a masterserver to both client and server, it showed up.

But when I tried to running masterserver from an another machine (which runs on the same network) and adding as a masterserver, the engine won't send packets to that masterserver and also it won't show up at the list. Same thing happens outside the local network

TL;DR 
If the masterserver hosted at the same computer with the server, it will show up in the list.
But if they are not, then it will not show up.

**Comment by ridvan1995** [comment]
With the old engine, everything works normally.

**Comment by a1batross** [comment]
Set `public 1` and `sv_lan 0`.
